packages/grafana-ui/src/components/Modal/Modal.tsx

```diff
@@ -40,6 +40,23 @@ export function Modal(props: PropsWithChildren<Props>): ReturnType<FC<Props>> {
     }
   }, [propsOnDismiss]);
 
+  const onEscKey = (ev: KeyboardEvent): void => {
```

Comment:
```suggestion
  const onEscKey = (ev: KeyboardEvent) => {
```
I think it's better to infer as much as possible, so it's less type maintenance.